### PR TITLE
pull data from https instead of fauna

### PR DESCRIPTION
**Pull request by j23414**
### Description of proposed changes

To match the zika workflow, pull data from https instead of fauna (rethinkdb).

### Related issue(s)

Related to pull https://github.com/nextstrain/zika/pull/15

### Testing

To test the full pipeline:

```
cd measles
nextstrain build . 
```

To test the smaller test dataset

```
cd measles
mkdir -p data
cp -v example_data/* data/.
nextstrain build . 
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,25 @@ python:
...
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  # Useful for debugging any issues with conda
+  - conda info -a
+  # Install nextstrain cli
+  - conda install -n base -c conda-forge mamba --yes
+  - conda activate base
+  - mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+  - conda activate nextstrain
+  - mamba install -c conda-forge -c bioconda augur auspice nextalign snakemake git --yes
```
Comment (victorlin) [inline .travis.yml:24]: With nextstrain/docs.nextstrain.org#102 the recommendation is to have just one `mamba create` vs. `mamba create` + `mamba install`:

```suggestion
  - mamba create -n nextstrain -c bioconda nextstrain-cli augur auspice nextalign snakemake git --yes
  - conda activate nextstrain
```

**Review by victorlin — CHANGES_REQUESTED** [review]

**Review by victorlin — APPROVED** [review]
Looks good! Would be good to clean up the git commit history with a `rebase`, or in the past I've used the **squash and merge** option on GitHub (but I don't think anyone else on the team prefers it). Let me know if you need help.

**Comment by j23414** [comment]
hmm, I might not have "merge" permissions for `nextstrain/measles`. Anyone else willing to merge?

**Comment by victorlin** [comment]
@j23414 I just added @nextstrain/core as a writer to this repo so you should be able to do so now.

**Review by tsibley — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -7,11 +7,24 @@ python:
...
+  # Useful for debugging any issues with conda
+  - conda info -a
+  # Install nextstrain cli
+  - conda install -n base -c conda-forge mamba --yes
+  - conda activate base
+  - mamba create -n nextstrain -c conda-forge -c bioconda nextstrain-cli augur auspice nextalign snakemake git --yes
+  - conda activate nextstrain
   - nextstrain version
   - nextstrain check-setup
   - nextstrain update
```
[the remark this answers is not in the document]
Comment (j23414) [reply .travis.yml:26]: From a "code smell" standpoint I agree with you @tsibley. The current expansion was a workaround until `pip3 install` was fixed for `.travis` testing. 

See https://github.com/nextstrain/zika/pull/14 for more details. Open other suggestions.

[the remark this answers is not in the document]
Comment (victorlin) [reply .travis.yml:26]: Suggestions:

1. Move to GitHub Actions where we're using [`conda-incubator/setup-miniconda@v2`](https://github.com/marketplace/actions/setup-miniconda) in repos like [augur](https://github.com/nextstrain/augur/blob/d9aac582d27180c6c0f3078f0450243b454a2bb5/.github/workflows/ci.yaml#L33)
2. Only do `mamba install nextstrain-cli`
3. Make it more clear that Docker is being used. Maybe `nextstrain check-setup --set-default`? Unless there is a way to do something like `nextstrain --set-runtime docker`

[the remark this answers is not in the document]
Comment (j23414) [reply .travis.yml:26]: I like option 1, to move toward a consistent CI across repos :)

[the remark this answers is not in the document]
Comment (tsibley) [reply .travis.yml:26]: Ah, I didn't see https://github.com/nextstrain/zika/pull/14. That's useful context.

I don't care too much about specifically conda/mamba vs. pip and Travis CI vs. GitHub Actions here, but good to be consistent when it makes sense (e.g. prefer GitHub Actions over Travis CI) and trim things down to minimal necessary steps.

Specifically here, I would wonder why did `pip install` start failing? Does it work now (i.e. was it transient)? It should work, and it's the simplest step (can be installed from latest PyPI release, doesn't have to be git and probably shouldn't be).

[the remark this answers is not in the document]
Comment (tsibley) [reply .travis.yml:26]: > Make it more clear that Docker is being used

I think adding the `--docker` flag to the `nextstrain build` invocation would suffice ± a comment.

[the remark this answers is not in the document]
Comment (j23414) [reply .travis.yml:26]: Lol, I think we were both taxing .travis test at the same time. 

* https://app.travis-ci.com/github/nextstrain/zika/builds 

As long as it's working now, I'm happy

**Pull request merged**